README.md

```diff
@@ -57,26 +57,33 @@
     ```
 
 4. Define Django model for index
-
     ```python
-
+    import six
```

Comment:
`six` - maybe should be imported from django.utils, because it is not present in package dependencies.